notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -92,7 +92,7 @@
     "                               productSubGroupDescription='FLC')\n",
     "\n",
     "# Move to working directory (not necessary, but outputs will all be in the same place if this is done)\n",
-    "input_flcs = glob.glob(os.path.join('mastDownload', 'HST', '*', '*flc.fits'))\n",
+    "input_flcs = sorted(glob.glob(os.path.join('mastDownload', 'HST', '*', '*flc.fits')))\n",
```

Comment:
If this is merely for copying files over and not actually used in Tweakreg, then sorting is not necessary?